### ISSUE TITLE
Suggestion ( MC 1.12.2 )

### ISSUE THREAD

**Issue by Kasia1403** [sp614x/optifine#7153]
To support Minecraft 1.12.2 and make clouds visible when render distance is lower than 4.

**Comment by Timolino456** [comment]
Uhm... what do you mean, there are OptiFine versions for 1.12.2, you simply have to go to optifine.net and click "show all versions"

**Comment by Kasia1403** [comment]
Yes, but in OptiFine clouds are visible when render distance is at least 4. But I would like that clouds will be visible when render distance is 2, because if I set more in my modpack my Minecraft world is so lagging that is unplayable. Is it possible to make an update for Minecraft 1.12.2 in which clouds will be visible when render distance is 2 ?
